ui/src/components/Publication/Creation/MainText.tsx

```diff
@@ -141,120 +142,72 @@ const MainText: React.FC = (): React.ReactElement | null => {
     const updateReferences = Stores.usePublicationCreationStore((state) => state.updateReferences);
     const user = Stores.useAuthStore((state) => state.user);
 
-    const [loading, setLoading] = React.useState(true);
-
-    React.useEffect(() => {
-        fetchAndSetReferences();
-        setLoading(false);
-    }, [content, user, publicationId]);
-
-    const fetchAndSetReferences = React.useCallback(async () => {
-        if (publicationId) {
-            try {
-                const response = await api.get(`/publications/${publicationId}/reference`, user?.token);
-                updateReferences(response.data);
-            } catch (err) {
-                /**
-                 * @TODO - handle errors - eg. cannot load references...
-                 */
-            }
+    const [loading, setLoading] = React.useState(false);
```

Comment:
This loading state is obsolete from what I saw. Same as const user = ... above. They are not used anymore